docs/reference-guide/streams/overview.md

```diff
@@ -140,3 +140,24 @@ the Kafka cluster. However, even though we cannot guarantee **exactly once**
 semantics, we tried to minimize the possibility of processing messages multiple
 times. This means committing the message offsets to the Kafka cluster happens
 right after the transaction is committed to the database.
+
+## Configuring stream transactions
+
+A stream can fail for various reasons. One of the failures of interest is when a transaction
+(in which the returned queries of the transformation are executed) fails to commit because of another conflicting 
+transaction. This is a side effect of [isolation levels](/reference-guide/isolation-levels.md) 
+and its behaviour can be configured by setting the appropriate memgraph flag:
+
+```
+--stream-transaction-conflict-retries=TIMES_TO_RETRY
+```
+
+By default, memgraph retries a transaction 31 times. If `TIMES_TO_RETRY` is set
+to 0 then memgraph tries a transaction only once. However, for any other value,
+memgraph retries with a total number of: 1 + `TIMES_TO_RETRY`. 
```

Comment:
```suggestion
By default, memgraph retries a transaction 31 times. If `TIMES_TO_RETRY` is set
to 0 then memgraph tries a transaction only once. However, for any other value,
memgraph retries with a total number of: 1 + `TIMES_TO_RETRY`. 
```
Weird phrasing of it. 31 seems like a strange number.
```
Memgraph will always try to execute the transaction once. 
If it fails because of conflicts with other transactions it will retry the transaction up to `TIMES_TO_RETRY` times. 
By default, that number is set to 30....
```